MediaPlayer.py

```diff
@@ -158,9 +164,30 @@ def read_track_metadata(self, track):
 		return tag
 
 	def search_tracks(self, **kwargs):
+		"""
+		Searches the MediaMonkey music library for tracks matching the artist and track title
+		:param kwargs:
+			See below
+
+		:keyword Arguments:
+			* *query*  (``str``)   -- MediaMonkey query string
+			* *rating* (``bool``)  -- Search for tracks that have a rating
+			* *title*  (``str``)   -- Search by Track title
+
+		:return: a list of matching tracks
+		:rtype: list<sync_items.AudioTag>
+		"""
+		title = kwargs.get('title')
+		rating = kwargs.get('rating')
+		query = kwargs.get('query')
+		if title: 
+			query = "SongTitle like \"%"+ title +"%\""
```

Comment:
I think something you might like are f-strings, e.g. 
```suggestion
			query = f'SongTitle like "%{title}%"'
```
also if you want to use `"`, you can use `'` to wrap the string